Lesson33/Deniz/lesson33-app/src/Exercise4/index.jsx

```diff
@@ -0,0 +1,28 @@
+import { useState } from "react";
+
+export const Exercise4 = () => {
+  const [items] = useState(["Apple", "Pear", "Banana", "Watermelon"]);
+  const [message, setMessage] = useState("");
+
+  const handleClick = (item) => {
```

Comment:
In this case, you don't need to reset the message to empty string, you can call `setMessage` function directly and provide a new value to it:
`setMessage(`Clicked on ${item}!`);`